app/controllers/api/v1/ecf_participants_controller.rb

```diff
@@ -41,19 +41,41 @@ def lead_provider
       end
 
       def participants
-        participants = lead_provider.ecf_participants
-                                    .distinct
-                                    .includes(
-                                      teacher_profile: {
-                                        current_ecf_profile: %i[cohort school ecf_participant_eligibility ecf_participant_validation_data participant_profile_state participant_profile_states schedule],
-                                        early_career_teacher_profile: :mentor,
-                                      },
-                                    )
-                                    .where(school_cohorts: { cohort_id: Cohort.current.id })
-
-        participants = participants.changed_since(updated_since) if updated_since.present?
-
-        participants.order(:created_at)
+        participant_profiles = ParticipantProfile::ECF.where(id: participant_profile_ids)
+                                                      .joins(:user)
+                                                      .includes(
+                                                        :user,
+                                                        :cohort,
+                                                        :school,
+                                                        :ecf_participant_eligibility,
+                                                        :ecf_participant_validation_data,
+                                                        :schedule,
+                                                        teacher_profile: { current_ect_profile: { mentor_profile: :user } },
+                                                      )
+
+        if updated_since.present?
+          participant_profiles = participant_profiles.where(user: { updated_at: updated_since.. })
+                                                     .order("\"user\".updated_at, \"user\".id")
+        end
+
+        participant_profiles.order("\"user\".created_at")
+      end
+
+      def participant_profile_ids
+        @participant_profile_ids ||= lead_provider.ecf_participant_profiles
+                                                  .select("DISTINCT ON (participant_profiles.teacher_profile_id) teacher_profile_id, participant_profiles.status, participant_profiles.id")
+                                                  .joins(:school_cohort)
+                                                  .where(school_cohort: { cohort_id: Cohort.current.id })
+                                                  .order(:teacher_profile_id, status: :asc)
+                                                  .map(&:id)
```

Comment:
```suggestion
                                                  .pluck(:id)
```

Reply:
This breaks with the distinct on I think, because pluck reSelects